### DISCUSSION TITLE
Cron Job Enhancement?

### DISCUSSION THREAD

**Discussion by wslyngstad** [alexjustesen/speedtest-tracker#1366] [Ideas]
Was wondering feasibility of updating cron scheduler in Speedtest Tracker to accommodate something like 3 2/4 * * *, so you can run a job every four hours starting at a particular time.  The cron generator link in the app shows the output for what I'd like to do, but Speedtest Tracker doesn't like the /4.  I get "Cron expression is not valid"

**Comment by alexjustesen** [comment]
#552

**Comment by wslyngstad** [comment]
thank you, I swear I searched and nothing came back.  3 2,6,10,14,18,22 * * * worked
